clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/FleetController.java

```diff
@@ -1215,10 +1219,10 @@ public void waitForNodesInSlobrok(int distNodeCount, int storNodeCount, int time
                 }
                 if (distCount == distNodeCount && storCount == storNodeCount) return;
 
-                long remainingTime = maxTime - System.currentTimeMillis();
-                if (remainingTime <= 0) {
+                Duration remainingTime = Duration.between(Instant.now(), endTime);
+                if (remainingTime.isNegative()) {
```

Comment:
Prefer Instant.now().isAfter(endTime) as above for consistency and readability.

Reply:
👍